tests/InstrEngineTests/InstrEngineTests/TestDynamicCode.cs

```diff
@@ -6,12 +6,19 @@
 namespace InstrEngineTests
 {
     [TestClass]
+    // "dotnet test" runs tests in-place (instead of copying them to a deployment
+    // directory). DeploymentItems that copy from a source path to a destination
+    // that is the same location are not necessary and cause file locking issues
+    // when initializing the tests. Only use DeploymentItem for paths with different
+    // destinations for .NET Core test assemblies.
+#if !NETCOREAPP
     [DeploymentItem(PathUtils.BaselinesBinPath, PathUtils.BaselinesBinPath)]
```

Comment:
>DeploymentItem [](start = 5, length = 14)

Is DeploymentItem an inheritable attribute? If yes, you could potentially just make a baseclass with all of these attributes attached instead of needing to repeat them.

Reply:
Unfortunately, it does not appear to be inheritable.